### PR TITLE
dz done

### PR DESCRIPTION
**Pull request by EKrivorog**

### CODE REVIEW

File: src/test/java/PersonTest.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+        evgeniy.registerPartnership(natalia);
+        evgeniy.deregisterPartnership(true);
+
+        assertNull(evgeniy.getPartner());
+        assertEquals("Krivorog", evgeniy.getLastName());
+    }
+    @Test
+    public void testIsRetiredForWoman() {
+        Woman natalia = new Woman("Natalia", "Krivorog", 60);
```
Comment (yurii-voronenko) [inline src/test/java/PersonTest.java:32]: Boundary values should be tested here, for woman it's 59, 60 ,61

File: src/test/java/PersonTest.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        assertTrue(natalia.isRetired());
+    }
+
+    @Test
+    public void testIsRetiredForMan() {
+        Man evgeniy = new Man("Evgeniy", "Krivorog", 65);
```
Comment (yurii-voronenko) [inline src/test/java/PersonTest.java:38]: Boundary values should be tested here, for woman it's 64, 65, 66

File: src/test/java/PersonTest.java [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    @Test
+    public void testDeregisterPartnership() {
+        Man evgeniy = new Man("Evgeniy", "Krivorog", 32);
+        Woman natalia = new Woman("Natalia", "Krivorog", 32);
+
+        evgeniy.registerPartnership(natalia);
+        evgeniy.deregisterPartnership(true);
+
+        assertNull(evgeniy.getPartner());
+        assertEquals("Krivorog", evgeniy.getLastName());
```
Comment (yurii-voronenko) [inline src/test/java/PersonTest.java:28]: Why asserts only for man but no for woman?

File: src/test/java/PersonTest.java [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+public class PersonTest {
+
+    @Test
+    public void testRegisterPartnership() {
+        Man evgeniy = new Man("Evgeniy", "Krivorog", 32);
+        Woman natalia = new Woman("Natalia", "Krivorog", 32);
+
+        evgeniy.registerPartnership(natalia);
+        assertEquals(natalia, evgeniy.getPartner());
+        assertEquals("Krivorog", evgeniy.getLastName());
```
Comment (yurii-voronenko) [inline src/test/java/PersonTest.java:16]: Very small amount of asserts, how about firstName, age for both woman and man. And also you do not check partner for woman and last name

**Review by yurii-voronenko — APPROVED** [review]
Done
